snakemake_conda/env.yaml

```diff
@@ -0,0 +1,514 @@
+  - smmap=4.0.0=pyhd3eb1b0_0
+  - snappy=1.1.10=h313beb8_1
+  - sniffio=1.3.0=py311hca03da5_0
+  - snowballstemmer=2.2.0=pyhd3eb1b0_0
+  - sortedcontainers=2.4.0=pyhd3eb1b0_0
+  - soupsieve=2.5=py311hca03da5_0
+  - sphinx=5.0.2=py311hca03da5_0
+  - sphinxcontrib-applehelp=1.0.2=pyhd3eb1b0_0
+  - sphinxcontrib-devhelp=1.0.2=pyhd3eb1b0_0
+  - sphinxcontrib-htmlhelp=2.0.0=pyhd3eb1b0_0
+  - sphinxcontrib-jsmath=1.0.1=pyhd3eb1b0_0
+  - sphinxcontrib-qthelp=1.0.3=pyhd3eb1b0_0
+  - sphinxcontrib-serializinghtml=1.1.5=pyhd3eb1b0_0
+  - spyder=5.4.3=py311hca03da5_1
+  - spyder-kernels=2.4.4=py311hca03da5_0
+  - sqlalchemy=2.0.25=py311h80987f9_0
+  - sqlite=3.41.2=h80987f9_0
+  - stack_data=0.2.0=pyhd3eb1b0_0
+  - statsmodels=0.14.0=py311hb9f6ed7_0
+  - streamlit=1.30.0=py311hca03da5_0
+  - sympy=1.12=py311hca03da5_0
+  - tabulate=0.9.0=py311hca03da5_0
+  - tapi=1100.0.11=h8754e6a_1
+  - tbb=2021.8.0=h48ca7d4_0
+  - tblib=1.7.0=pyhd3eb1b0_0
+  - tenacity=8.2.2=py311hca03da5_0
+  - terminado=0.17.1=py311hca03da5_0
+  - text-unidecode=1.3=pyhd3eb1b0_0
+  - textdistance=4.2.1=pyhd3eb1b0_0
+  - threadpoolctl=2.2.0=pyh0d69192_0
+  - three-merge=0.1.1=pyhd3eb1b0_0
+  - tifffile=2023.4.12=py311hca03da5_0
+  - tinycss2=1.2.1=py311hca03da5_0
+  - tk=8.6.12=hb8d0fd4_0
+  - tldextract=3.2.0=pyhd3eb1b0_0
+  - toml=0.10.2=pyhd3eb1b0_0
+  - tomlkit=0.11.1=py311hca03da5_0
+  - toolz=0.12.0=py311hca03da5_0
+  - tornado=6.3.3=py311h80987f9_0
+  - tqdm=4.65.0=py311hb6e6a13_0
+  - traitlets=5.7.1=py311hca03da5_0
+  - truststore=0.8.0=py311hca03da5_0
+  - twisted=23.10.0=py311hca03da5_0
+  - typing-extensions=4.9.0=py311hca03da5_1
+  - typing_extensions=4.9.0=py311hca03da5_1
+  - tzdata=2023d=h04d1e81_0
+  - tzlocal=2.1=py311hca03da5_1
+  - uc-micro-py=1.0.1=py311hca03da5_0
+  - ujson=5.4.0=py311h313beb8_0
+  - unidecode=1.2.0=pyhd3eb1b0_0
+  - unixodbc=2.3.11=h1a28f6b_0
+  - urllib3=2.0.7=py311hca03da5_0
+  - utf8proc=2.6.1=h80987f9_1
+  - validators=0.18.2=pyhd3eb1b0_0
+  - w3lib=2.1.2=py311hca03da5_0
+  - watchdog=2.1.6=py311h80987f9_0
+  - wcwidth=0.2.5=pyhd3eb1b0_0
+  - webencodings=0.5.1=py311hca03da5_1
+  - websocket-client=0.58.0=py311hca03da5_4
+  - werkzeug=2.2.3=py311hca03da5_0
+  - whatthepatch=1.0.2=py311hca03da5_0
+  - wheel=0.41.2=py311hca03da5_0
+  - widgetsnbextension=3.5.2=py311hca03da5_1
+  - wrapt=1.14.1=py311h80987f9_0
+  - wurlitzer=3.0.2=py311hca03da5_0
+  - xarray=2023.6.0=py311hca03da5_0
+  - xlwings=0.29.1=py311hca03da5_0
+  - xyzservices=2022.9.0=py311hca03da5_1
+  - xz=5.4.5=h80987f9_0
+  - yaml=0.2.5=h1a28f6b_0
+  - yaml-cpp=0.8.0=h313beb8_0
+  - yapf=0.31.0=pyhd3eb1b0_0
+  - yarl=1.9.3=py311h80987f9_0
+  - zeromq=4.3.5=h313beb8_0
+  - zfp=1.0.0=h313beb8_0
+  - zict=3.0.0=py311hca03da5_0
+  - zipp=3.17.0=py311hca03da5_0
+  - zlib=1.2.13=h5a0b063_0
+  - zlib-ng=2.0.7=h80987f9_0
+  - zope=1.0=py311hca03da5_1
+  - zope.interface=5.4.0=py311h80987f9_0
+  - zstandard=0.19.0=py311h80987f9_0
+  - zstd=1.5.5=hd90d995_0
```

Comment:
What is the purpose of the numbers after the version number?

Reply:
They can and arguably should all be removed. (I normally do so after exporting the environment.) The important part is showing and locking down the exact version that was used.

Reply:
removed numbers after version number in the updated environment.yaml file.